### PR TITLE
Smaato: Add ORTB2.6 tests

### PR DESCRIPTION
**Pull request by Enigo**
We are in process of migration to ortb26 so in order to make sure no changes needed on the prebid-server side we've added more tests here

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,11 +43,8 @@ inventory_url.yaml
 analytics/config/testFiles/
 analytics/filesystem/testFiles/
 
-# autogenerated version file
-# static/version.txt
-
-.idea/
-.vscode/
+# autogenerated files
+prebid-server.iml
```
Comment (onkarvhanumante) [inline .gitignore:47]: @Enigo any particular reason for this change. Else could discard this file change

Comment (Enigo) [reply .gitignore:47]: cleaned it up a bit:
* `.idea/` was duplicated
* `.vscode/` should be up there with the other folders
* `iml` file is a default IDEA file that should be versioned

Comment (Enigo) [reply .gitignore:47]: @onkarvhanumante 
how does it sound to you?

Comment (Enigo) [reply .gitignore:47]: or maybe @Sonali-More-Xandr ?

Comment (onkarvhanumante) [reply .gitignore:47]: > cleaned it up a bit:
> 
> * `.idea/` was duplicated
> * `.vscode/` should be up there with the other folders
> * `iml` file is a default IDEA file that should be versioned

how does above change relates to `Smaato: Add ORTB2.6 tests`

Comment (Enigo) [reply .gitignore:47]: it is not related, but the change isn't significant enough to have a dedicated PR, don't you think?

Comment (onkarvhanumante) [reply .gitignore:47]: Generally its good to isolate 2 different changes in seperate PRs and include details in PR description.

In this case, description was missing for `.idea/` `.vscode/` `iml`

File: adapters/smaato/smaato.go [hunk 2]
```diff
@@ -545,20 +533,6 @@ func makeImpExt(impExtRaw *json.RawMessage) (json.RawMessage, error) {
 	}
 }
 
-func setBannerDimension(banner *openrtb2.Banner) (*openrtb2.Banner, error) {
```
Comment (onkarvhanumante) [inline adapters/smaato/smaato.go:548]: @Enigo  any particular reason to remove `setBannerDimension` >

Comment (Enigo) [reply adapters/smaato/smaato.go:548]: Bcz it's not needed, server-side we can process w/h from both banner and banner.format, so there's no need to duplicate them.

File: .gitignore (hunk 1 above)
Comment (Enigo) [reply .gitignore]: alright, the file was reverted

**Review by onkarvhanumante — APPROVED** [review]

**Comment by Enigo** [comment]
@Sonali-More-Xandr 
could you please take a look as well?
thanks!

File: adapters/smaato/smaato.go (hunk 2 above)
Comment (Enigo) [reply adapters/smaato/smaato.go:548]: @onkarvhanumante 
can this PR be merged with just one approve from you? 

@Sonali-More-Xandr seems to be unavailable

**Review by el-chuck — APPROVED** [review]

**Comment by Enigo** [comment]
@gargcreation1992 
could you please take a look at this PR?
thanks!

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**